### PR TITLE
[Service Bus] Add MinTls to `az servicebus namespace create/update` and add `az servicebus namespace network-rule update`

### PR DESCRIPTION
**Pull request by v-Ajnava**
**Related command**

az servicebus namespace create 
az servicebus namespace update 

**Description**

PR is to add support for Min TLS version at namespace level. 
Added parameter to able to set Min TLS version for given namespace

**Testing Guide**

**History Notes**

[Service Bus] `az servicebus namespace create/update`: Support specifying `--min-tls`
[Service Bus] `az servicebus namespace network-rule update`: Support updating network rules for given namespace

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [X] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Service Bus

**Comment by v-Ajnava** [comment]
Hi @evelyn-ys , can you please review the PR ?

File: src/azure-cli/azure/cli/command_modules/servicebus/_help.py [hunk 1]
```diff
@@ -272,6 +272,14 @@
     text: az servicebus namespace network-rule list --resource-group myresourcegroup --namespace-name mynamespace
 """
 
+helps['servicebus namespace network-rule update'] = """
+type: command
+short-summary: Updates network rule properties of the given Namespace.
+examples:
+  - name: Updates network rule properties of the given Namespace, can be used to update public network access, trusted service and default action.
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/servicebus/_help.py:279]:
```suggestion
short-summary: Update network rule properties of the given Namespace.
examples:
  - name: Update network rule properties of the given Namespace, can be used to update public network access, trusted service and default action.
```

**Review by evelyn-ys — APPROVED** [review]
LGTM in general

**Review by necusjz — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

File: src/azure-cli/azure/cli/command_modules/servicebus/_help.py (hunk 1 above)
Comment (v-Ajnava) [reply src/azure-cli/azure/cli/command_modules/servicebus/_help.py]: changes pushed

**Pull request merged**